offworld_gym/envs/gazebo_docker/remote_env.py

```diff
@@ -26,6 +26,9 @@
 from offworld_gym.envs.gazebo_docker.protobuf.remote_env_pb2_grpc import RemoteEnvStub
 
 logger = logging.getLogger(__name__)
+level = logging.DEBUG
+# logger.setLevel(level)
```

Comment:
If this line is not doing anything please remove

Reply:
This line can be comment out, since it is debugging level flag, in case someone wants to switch to debug mode